zbsprimarystorage/zbsprimarystorage/zbsutils.py

```
@@ -66,7 +73,8 @@ def get_version():
 
 
 def deploy_client(ip, port, username, password):
-    return shell.call("%s client deploy --host %s --port %s -u %s -p %s --silent" % (ZBSADM_BIN_PATH, ip, port, username, linux.shellquote(password)))
+    return shell.call("%s client deploy --host %s --port %s -u %s -p %s --silent" % (
```

Ruff (0.13.3)

`shell` may be undefined, or defined from star imports

(F405)